frontend/src/query/mutations.ts

```diff
@@ -18,13 +19,9 @@ interface CreateAdmissionProps {
 
 export const useAdminCreateAdmission = () => {
   const queryClient = useQueryClient();
-  return useMutation(
-    ({ admission }: CreateAdmissionProps) => {
-      return callApiFromQuery(`/admin/admission/`, {
-        method: "POST",
-        body: JSON.stringify(admission),
-      });
-    },
+  return useMutation<Record<string, unknown>, AxiosError, CreateAdmissionProps>(
```

Comment:
Do we not know the type of `Record<string, unknown>`? :thinking:

Reply:
Very good point, we do actually